Source/Core/Core/IOS/Network/KD/NetKDRequest.cpp

```diff
@@ -691,6 +692,107 @@ IPCReply NetKDRequestDevice::HandleNWC24DownloadNowEx(const IOCtlRequest& reques
   return IPCReply(IPC_SUCCESS);
 }
 
+IPCReply NetKDRequestDevice::HandleRequestRegisterUserId(const IOS::HLE::IOCtlRequest& request)
```

Comment:
All of the return values from this function are IPCReply{IPC_SUCCESS}. Unless that's going to change in the future this function should return void and the caller should return that value explicitly in order to reduce the amount of code people need to read to understand where a given return value is coming from.

Reply:
I am not entirely sure how this would react, it would be replying to IPC twice like this.

Reply:
In the future I will probably refactor the async handler as majority of the functions don't reply anything but `IPC_SUCCESS`

Reply:
I misunderstood what was happening here, go ahead and ignore that suggestion.